docs/backends/Google.md

```diff
@@ -202,6 +202,33 @@ If the Docker image to be pulled is not public the `docker pull` will fail which
 If using any of these private Docker workflow options it is advisable to add
 them to the `workflow-options.encrypted-fields` list in Cromwell configuration.
 
+**Runtime Compute Engine configuration**  
+_runtime_ section of the WDL task contains information about the required computational resources.
+```
+runtime {
+    cpu: CPU_COUNT
+    memory: MEM_SIZE
+    cpuPlatform: PLATFORM
+  }
+```
+By default Cromwell creates custom type instances:  
+>{PREFIX}custom-X-Y
+where
+* PREFIX indicates generation: "n2-" for Cascade Lake CPUs, empty for others
+* X - vCPUs count
+* Y - memory size (MB)  
+
+You can use predefined [machine type](https://cloud.google.com/compute/docs/machine-types) instances using `machineType` tag.
```

Comment:
It [seems](https://cromwell.readthedocs.io/en/stable/RuntimeAttributes/) that this is not a runtime *tag*, but a runtime *attribute*.